### PR TITLE
#156739693: Provide explanation of collateralization

### PR DESCRIPTION
**Pull request by saturnial**

### CODE REVIEW

File: src/modules/RequestLoan/RequestLoanForm/schema.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { JSONSchema4 } from "json-schema";
+const singleLineString = require("single-line-string");
```
Comment (chrismin) [inline src/modules/RequestLoan/RequestLoanForm/schema.tsx]: We should generally tend towards the `import` syntax:

`import singleLineString from "single-line-string";`

File: src/modules/RequestLoan/RequestLoanForm/schema.tsx [hunk 2]
```diff
@@ -63,13 +64,17 @@ export const schema: JSONSchema4 = {
...
                     type: "boolean",
                     title: "Collateralized",
-                    description:
-                        "A quick, layman's definition of what collateralized means and why it's a smart idea goes here.",
+                    description: singleLineString`Collateralization occurs when you, the borrower,
+                    pledge an asset as recourse to the lender in the event that you, the borrower,
+                    default on the original loan. Collateralization of assets gives lenders a
+                    sufficient level of reassurance against default risk. It also helps you, the
+                    borrower, receive loans that you might not otherwise receive given your credit
+                    history or lack thereof.`,
```
Comment (nadavhollander) [inline src/modules/RequestLoan/RequestLoanForm/schema.tsx]: This language, though extremely eloquent and accurate, is a bit too high vocab for our target audience IMO.  Please make briefer and simpler in language.

File: src/modules/RequestLoan/RequestLoanForm/schema.tsx (hunk 1 above)
Comment (saturnial) [reply src/modules/RequestLoan/RequestLoanForm/schema.tsx]: using the import syntax you suggest results in the following:

`could not find a declaration file for module: 'single-line-string'.`

File: src/modules/RequestLoan/RequestLoanForm/schema.tsx (hunk 2 above)
Comment (saturnial) [reply src/modules/RequestLoan/RequestLoanForm/schema.tsx]: I've updated the wording to be more low-brow ;)

**Comment by saturnial** [comment]
♻️ @nadavhollander @chrismin

**Pull request merged**